packages/orchestration/src/orchestration-types.ts

```diff
@@ -437,30 +438,36 @@ interface TranslationConfigParametersInput {
 }
 
 /**
- * Output parameters for translation configuration.
+ * Input parameters for translation input configuration.
  */
-interface TranslationConfigParametersOutput {
+export type TranslationInputParameters = TranslationConfigParametersInput & {
   /**
-   * Language of the text to be translated.
-   * @example sourceLanguage: 'de-DE'
+   * If true, the messages history will be translated as well.
+   * @example translateMessagesHistory: true
    */
-  sourceLanguage?: string;
+  translateMessagesHistory?: boolean;
   /**
-   * Language to which the text should be translated.
-   * @example targetLanguage: 'en-US'
+   * Configuration for applying translation to specific placeholders or message roles.
+   * @example applyTo: [{ category: 'placeholders', items: ['user_input'], source_language: 'de-DE' }]
    */
-  targetLanguage: string;
-}
+  applyTo?: SAPDocumentTranslationApplyToSelector[];
+};
 
 /**
  * Output parameters for translation output configuration.
  */
-export type TranslationOutputParameters = TranslationConfigParametersOutput;
-
-/**
- * Input parameters for translation input configuration.
- */
-export type TranslationInputParameters = TranslationConfigParametersInput;
+export interface TranslationOutputParameters {
+  /**
+   * Language of the text to be translated.
+   * @example sourceLanguage: 'de-DE'
+   */
+  sourceLanguage?: string;
+  /**
+   * Configuration for applying translation to specific placeholders or message roles, or a target language string.
+   * @example targetLanguage: { category: 'placeholders', items: ['assistant_response'] }
+   */
+  targetLanguage?: string | SAPDocumentTranslationApplyToSelector;
+}
```

Comment:
[req] Previously, we had an internal interface, `TranslationConfigParametersOutput` which was exposed directly via type `TranslationOutputParameters`. 
Please revert to the old state, and also make `targetLanguage` a required field as making this optional is a **Breaking change.** 

Please always review your changes, they should never include breaking changes in minor version releases, exceptions are possible in generated clients, but even then we include compatibility notes.